### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by sindresorhus**
This fixes a bunch of typos and some other minor tweaks.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 *.log
-.DS_Store
```
Comment (sindresorhus) [inline .gitignore:2]: This should be in your own global gitignore, not here, as it's not project-specific.

File: readme.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ Draqula is a GraphQL client for React apps that don't need everything.
 Instead of offering tons of features like extensive caching mechanism, local state management, subscriptions and so on, Draqula focuses on executing the basics well - queries and mutations.
 
 If you want to take a quick look, keep scrolling this readme.
-Otherwise, check out documentation at https://draqulajs.com.
+Otherwise, check out the documentation at https://draqulajs.com.
 
 ## Install
 
-```bash
+```console
```
Comment (sindresorhus) [inline readme.md:17]: It's not valid Bash because of the `$`.

File: tsconfig.json [hunk 3]
```diff
@@ -10,12 +10,6 @@
 		"rootDir": "./src",
 		"outDir": "./dist",
 		"strict": true,
-		"noImplicitAny": true,
-		"strictNullChecks": true,
-		"strictFunctionTypes": true,
-		"strictPropertyInitialization": true,
-		"noImplicitThis": true,
-		"alwaysStrict": true,
```
Comment (sindresorhus) [inline tsconfig.json:18]: These are all moot as `strict` already turns them on.

File: .gitignore (hunk 1 above)
Comment (felipeorlando) [reply .gitignore:2]: Not of all contributors could be this on global .gitignore. My suggest is to remain this.

Comment (sindresorhus) [reply .gitignore:2]: Then they should. Adding it here is a slippery slope to add lots of random files. Next there’s `.idea`, `.vscode`, etc.

Comment (felipeorlando) [reply .gitignore:2]: I agree with you, but we're talking about an open source project, that **anyone** can contribute. Do you trust in anyone?

Comment (vadimdemedes) [reply .gitignore:2]: I agree with @sindresorhus, I wouldn't want to accept .gitignore PRs for all the possible tools.

**Review by vadimdemedes — APPROVED** [review]

**Comment by vadimdemedes** [comment]
This is amazing, thank you Sindre!

File: docs/src/pages/docs/queries.mdx [hunk 4]
```diff
@@ -248,22 +248,22 @@ const Todos = () => {
 
 			{data.todos.pagination.hasMore && (
 				<button disabled={isFetchingMore} onClick={onFetchMore}>
-					{isFetchingMore ? 'Loading more todos...' : 'Load more todos'}
+					{isFetchingMore ? 'Loading more todos…' : 'Load more todos'}
```
Comment (shreyasminocha) [inline docs/src/pages/docs/queries.mdx:251]: I like the attention to detail

**Pull request merged**